test_scripts/API/AddCommand/ATF_P_AddCommand_UI_AddCommand_WARNINGS.lua

```diff
@@ -0,0 +1,134 @@
+--
+-- Expected result:
+-- SDL -> MOB: appID: (WARNINGS, success: true: AddCommand)
+---------------------------------------------------------------------------------------------
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.SDLStoragePath = config.pathToSDL .. "storage/"
+-- ToDo (vvvakulenko): remove after issue "ATF does not stop HB timers by closing session and connection" is resolved
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+
+--[[ Local Variables ]]
+local storagePath = config.SDLStoragePath..config.application1.registerAppInterfaceParams.appID.. "_" .. config.deviceMAC.. "/"
+local ServerAddress = commonFunctions.read_parameter_from_smart_device_link_ini("ServerAddress") 
+local grammarIDValue
+
+--[[ General Precondition before ATF start ]]
+commonFunctions:SDLForceStop()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivationApp()
+  local request_id = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
+  EXPECT_HMIRESPONSE(request_id)
+  :Do(function(_,data)
+    if (data.result.isSDLAllowed ~= true) then
+      local request_id1 = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+      EXPECT_HMIRESPONSE(request_id1)
+      :Do(function(_,_)   
+        self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = ServerAddress}})
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,_)
+          self.hmiConnection:SendResponse(data.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+        :Times(1)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"}) 
+end
+
+commonSteps:PutFile("Precondition_PutFile", "icon.png")
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_AddCommand_VR_warning()
+  local cid = self.mobileSession:SendRPC("AddCommand",
+  {
+    cmdID = 11,
+    menuParams =  
+    { 
+      parentID = 0,
+      position = 0,
+      menuName ="Commandpositive"
+    }, 
+    vrCommands = 
+    { 
+      "VRCommandonepositive",
+      "VRCommandonepositivedouble"
+    }, 
+    cmdIcon =   
+    { 
+      value ="icon.png",
+      imageType ="STATIC"
+    }
+  })
+  EXPECT_HMICALL("UI.AddCommand", 
+  { 
+    cmdID = 11,
+    cmdIcon = 
+    {
+      value = storagePath .."icon.png", 
```

Comment:
@vvvakulenko, SDL can return absolute path for this and you should check that value is sub-string of the full SDL path that is returned.
Please make verification of 'value' to be checked as sub-string, because at the moment the test has an error:
Exp: ./SDL_bin/storage/0000001_12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0/icon.png
Real: /home/istoimenova/SmartDeviceLinkCore/test_run/SDL_bin/storage/0000001_12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0/icon.png

You can use as example:
if (string.match(data.params.cmdIcon.value, "%S*" .. "("..string.sub(storagePath, 2).."icon.png)" .. "$") == nil ) then
-- print some error
end